src/eureka/S3_data_reduction/optspex.py

```diff
@@ -535,7 +535,7 @@ def clean_median_flux(data, meta, log, m):
 
         # Flag outliers
         outliers = sigma_clip(residuals, sigma=meta.median_thresh, maxiters=5,
-                              axis=1, cenfunc='median')
+                              axis=1, cenfunc=np.ma.median, stdfunc=np.ma.std)
```

Comment:
Looking at Figure 3401, this change doesn't have the desired effect for me (the entire region is yellow).  Please look at your output and verify the results.

Reply:
I had figured it'd be important to use those functions given we're using masked arrays, but it does seem to end up giving a significantly less smoothed median frame for me. I don't get the same behaviour as you (I'm using MIRI data), but I do get a worse median frame. However, the 2D LC with `cenfunc='median'` is far worse as it seems the sharp curvature in spectral response ends up getting partially clipped due to the smoothing (I used `window_len=11`). But either way, I've found that not smoothing works better for MIRI and if this change makes things worse for you, then I'll revert it